assets/typescript/authentication/authentication.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import QrCode from "./components/qrcode";
+import CheckSession from "./components/check-session";
 
 declare const image: string;
+declare const timeOutMessage: string;
 
 ReactDOM.render(
-    <QrCode
-        image={image}
-    />,
+    <div>
+        <QrCode
+            image={image}
+        />
+        <CheckSession
+          verifyUrl="/verify"
+          authenticationUrl="/authentication"
+          retryInSeconds={3}
+          maxTries={40}
```

Comment:
According to the PM this would suffice for now. We could always make this configurable later on, but then some ample input validation needs to be implemented.